clang-tools-extra/docs/clang-tidy/checks/abseil/string-find-startswith.rst

```diff
@@ -28,9 +28,9 @@ Options
 
 .. option:: StringLikeClasses
 
-   Semicolon-separated list of names of string-like classes. By default only
-   ``std::basic_string`` is considered. The list of methods to considered is
-   fixed.
+   Semicolon-separated list of names of string-like classes. By default both
```

Comment:
Line 6 in this file should also be changed, something like:
"Checks whether a ``std::string::find()`` or ``std::string::rfind()``  (and corresponding ``std::string_view`` methods)"

Simply should be there some info also about string_view, no need to update examples, just description.